docs/source/features/sheet_management/dataframe.rst

```diff
@@ -111,6 +114,45 @@ Extend another Dataframe to the end of the DataFrame:
    df1 = DataFrame([{"filename": "a.jpg"}])
    df.extend(df1)
 
+Add Columns
+===========
```

Comment:
add some complex examples, FileSeries, ArraySeries? At that time, users need to construct a series first